NERODesign/content/OffScreen.qml

```diff
@@ -0,0 +1,303 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+Item {
+    width: 1024
+    height: 768
+    visible: true
+
+    GridLayout {
+        id: gridLayout
+        x: parent.width * 0.3
+        y: parent.height * 0.15
+        width: parent.width * 0.65
+        height: parent.height * 0.8
+        rows: 6
+        columns: 2
+        columnSpacing: 10
+        rowSpacing: 10
+
+        // DetailDisplay elements
+        DetailDisplay {
+            Layout.row: 0
+            Layout.column: 0
+            Layout.columnSpan: 1
+            Layout.rowSpan: 1
+            width: parent.width * 0.45
+            height: parent.height * 0.1
+            shutdownFlowTask: "SIDE BRBs"
```

Comment:
the design looks good, you also need to copy this to development and hook up the controller.